pkg/model/sdk_helper.go

```diff
@@ -382,20 +384,27 @@ func (a *SDKAPI) SDKAPIInterfaceTypeName() string {
 }
 
 // Override the operation type and/or resource name if specified in config
-func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) (OpType, string) {
+func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) ([]OpType, string) {
 	opType, resName := GetOpTypeAndResourceNameFromOpID(opID)
-
+	opTypeArray := make([]OpType, 0)
+	// TODO: or use var opTypeArray []OpType
 	if cfg != nil {
 		if operationConfig, exists := cfg.Operations[opID]; exists {
-			if operationConfig.OperationType != "" {
-				opType = OpTypeFromString(operationConfig.OperationType)
-			}
-
 			if operationConfig.ResourceName != "" {
 				resName = operationConfig.ResourceName
```

Comment:
This call now overrides `resName` every iteration. Maybe panic if the resource name changes?

Reply:
good point, I assumed a valid config.

Reply:
On second thought, each opID will always have only a single resName so there is no overriding happening here, so a panic is unnecessary. Am I missing some case ?

Reply:
My point is that, right now we are assuming that all operations have the same `resName`,which is most likely correct. However just in case, we need to panic if we receive multiple operations for different `resName`.